controller/controller.go

```diff
@@ -203,15 +210,22 @@ type Impl struct {
 
 	// StatsReporter is used to send common controller metrics.
 	statsReporter StatsReporter
+
+	// GlobalResyncFilterFunc is used to filter our objects from
+	// the shared cache on a global resync, be default and if not
+	// set by the controller implemenation, it will default to
+	// allowing every object in the cache
```

Comment:
```suggestion
	// GlobalResyncFilterFunc is used to filter out objects from
	// the shared cache on a global resync. By default and if not
	// set by the controller implemenation, it will default to
	// allowing every object in the cache.
```